app/models/domain/fim_measurements.rb

```diff
@@ -1,3 +0,0 @@
-class Domain::FimMeasurements < ActiveRecord::Base
```

Comment:
It occurs to me that we should actually restore this file.  I think there is both a domain_fim_measurements and a fim_measurements table.